nbextensions/bulkImportCell/tabs/xsvGenerator.js

```diff
@@ -274,107 +273,99 @@ define(['StagingServiceClient', 'common/html', 'common/runtime', 'common/ui', 'u
-                                    );
-                                }
-                            )
-                        ),
-                    ]),
-
-                    // output file directory
-                    fieldset([
-                        p(
-                            label(
-                                {
-                                    for: formConfig.specs.output_directory.original.id,
-                                },
-                                formConfig.specs.output_directory.original.ui_name
-                            )
-                        ),
-                        input(
-                            {
-                                id: formConfig.specs.output_directory.original.id,
-                                type: 'text',
-                                name: formConfig.specs.output_directory.original.id,
-                                placeholder:
-                                    formConfig.specs.output_directory.original.text_options
-                                        .placeholder,
-                                value: formConfig.specs.output_directory.original.text_options
-                                    .placeholder,
-                                minlength: 1,
-                                required: true,
-                            },
-                            ''
-                        ),
-                    ]),
+                    div({
+                        id: this.id,
+                    }),
                 ]
             );
         }
 
+        /**
+         * This shows the parameters widget - the inputs for the singleton set of parameters
+         * that get applied to every job in the bulk run.
+         * @param {DOMElement} node the container node for the widget
+         * @returns {object} keys:
+         *   bus - the message bus created for this widget
+         *   instance - the created widget
+         */
+        buildParamsWidget(node) {
+            this.paramsBus = this.runtime
+                .bus()
+                .makeChannelBus({ description: 'Parent comm bus for parameters widget' });
+
+            this.paramsBus.on('parameter-changed', (message) => {
+                this.updateModelParameterValue(message);
+            });
+
+            this.paramsWidget = ParamsWidget.make({
+                bus: this.paramsBus,
+                workspaceId: this.runtime.workspaceId(),
+                paramIds: this.internalModel.getItem(['inputs', 'otherParamIds']),
+                initialParams: this.internalModel.getItem(['params']),
+                initialDisplay: {},
+            });
+
+            return this.paramsWidget.start({
+                node,
+                parameters: this.spec.getSpec().parameters,
+            });
+        }
+
+        startInputWidgets() {
+            this.paramsNode = document.createElement('div');
+            return Promise.try(() => {
```

Comment:
`this.buildParamsWidget` already returns a Promise, don't need to wrap it!

Reply:
Fixed